ib_fabric.md

```diff
@@ -1,11 +1,28 @@
-# Infaband Fabric
+# InfiniBand Fabric
 
-The following information is for the Infaband (IB) Fabrics hosted within the lab.
+The following information is for the InfiniBand (IB) Fabrics hosted within the lab.
 
 ## Switch & Fabric
 
-TBD
+The IB fabric is created with the Mellanox Quantum QM8700 HDR IB 40 Port Switch.
+
+In the port definitions below, the power numbers refer to the 40G / 100G port. When
+that port is connected to a breakout cable, either to 10G or 25G, the physical
+sub-port is indicated as the second number.  *Logical ports will be updated
+when available.*
 
 ## Test Nodes
 
-TBD
+The following test nodes have **Mellanox MCX653106A-HDAT 200G 2-port** cards installed in PCI Slot 3:
+
+* node-07.ofa.iol.unh.edu - Connected to switch port 1 *and Ethernet switch port 8*
```

Comment:
Can we reference the specific switches names / models, so it's more clear what is connected to what.